src/Tiles.cpp

```diff
@@ -229,7 +229,7 @@ byte_vec_t Tileset::native_data() const {
 std::vector<Tile> Tileset::remap_tiles_for_output(const std::vector<Tile>& tiles, Mode mode) const {
   std::vector<Tile> tv;
 
-  if ((mode == Mode::snes && (_tile_width == 16 || _tile_height == 16))
+  if ((mode == Mode::snes && (_tile_width%16 == 0 || _tile_height%16 == 0) && (_tile_width <= 64 || _tile_height <= 64)) 
```

Comment:
This allows 48x48 sprites, which is [invalid](https://problemkaputt.de/fullsnes.htm#snesppuspritesobjs).